examples/op-stack-flashblocks.md

```diff
@@ -0,0 +1,24 @@
+# Op Stack Rollup Boost with Flashblocks
+
+This example shows how to deploy an Op Stack with rollup-boost using Flashblocks for pre-confirmations.
+
+## What is Flashblocks?
+
+Flashblocks is a feature in rollup-boost that enables pre-confirmations by proposing incremental sections of blocks. This allows for faster transaction confirmations and improved user experience on L2 networks.
+
+## Quick Start
+
+Deploy the Op Stack with rollup-boost and Flashblocks enabled:
+
+```bash
+$ go run main.go cook opstack --external-builder op-rbuilder --flashblocks
+```
+
+This will deploy an Op Stack chain with:
+
+- A complete L1 setup (CL/EL/Mev-boost)
```

Comment:
```suggestion
- A complete L1 setup (CL/EL/MEV-Boost)
```

Reply:
Removing it since it does not run Mev-boost this setup.